app/templates/components/article-featured-video.hbs

```diff
@@ -0,0 +1,17 @@
+<div class="video-container{{unless isPlayerLoading ' player-ready'}}{{if isPlayed ' is-played'}}"
+	 style="background-image: linear-gradient(to top, rgba(0, 0, 0, 0.9) 0%, transparent 100%), url({{model.url}});">
```

Comment:
I'm aware of that but that PR https://github.com/Wikia/mobile-wiki/pull/83 will fix it. So I left it as is.